source/includes/_reference.md

```diff
@@ -131,7 +131,7 @@ def enable_scoutprof?; true; end
 
 ### Free Trial
 
-We offer a no-risk, free 14-day trial. Delete your monitored apps before the trial concludes and you don't pay.
+We offer a no-risk, fully featured, free 14-day trial. Enter a credit or debit card to begin your subscription.
```

Comment:
We don't really need to mention the whole second sentence here I think. It's a free trial, at the end, of course it stops being free.

Reply:
Hmmm, makes sense. At the very least though, we should remove the original second sentence.

Reply:
```suggestion
We offer a no risk, fully featured, free trial. Enter a credit or debit card anytime to continue using Scout APM after the end of your trial.
```

I suggest this because some folks sometimes get longer trials, and we may be doing more trial experimentation in the future - it'd be better if we don't have to remember to update our docs as we adjust the trial length. 

I've also suggested an adjusted second sentence. Keep it if you like, or not :)

Reply:
The 14-day trial is standard and is shown on the home page. If we're advertising that we're flexible on trial length, where would we put that information?
![Image 2019-04-24 at 9 08 30 AM](https://user-images.githubusercontent.com/26899691/56675376-c6d92280-6670-11e9-8e6b-ca0fb2f57ff7.png)

Reply:
We are not advertising that we are flexible on trial length. Instead, my request is a change that makes the documentation less tightly coupled to the business terms. Not including the length of the trial in the docs doesn't advertise that our trials are flexible - but it does make it easier, in the future, to have trials of different lengths. A minor optimization, no big deal.